Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -88,6 +88,10 @@ public struct LocalizationKey {
     public static let cardNumberItemUnknownBrand = LocalizationKey(key: "adyen.card.numberItem.unknownBrand")
     /// Scan your card
     public static let cardScanYourCardButton = LocalizationKey(key: "adyen.card.scanYourCardButton")
+    /// Card Brand
+    public static let creditCardDualBrandTitle = LocalizationKey(key: "adyen.creditCard.dualBrand.title")
```

Comment:
This feature introduces semantic split for same feature:
- dual brand
- co badged card

I suggest we pick one and stick to one. Otherwise it becomes incredibly hard to connect all dots and build a mindmap later when making any changes.